opencti-platform/opencti-graphql/src/modules/notification/notification-domain.ts

```diff
@@ -48,22 +50,19 @@ const extractUniqRecipient = async (
   user: AuthUser,
   triggerInput: TriggerDigestAddInput | TriggerLiveAddInput,
   type: TriggerType
-): Promise<{ id: string, name: string }> => {
-  if (triggerInput.recipients && triggerInput.recipients.length > 0) {
+): Promise<BasicStoreEntity> => {
+  const { recipients } = triggerInput;
+  let recipient = user.id;
+  if (recipients?.at(0)) {
     if (!isUserHasCapability(user, SETTINGS_SET_ACCESSES)) {
       throw ForbiddenAccess();
     }
-    if (triggerInput.recipients.length > 1) {
+    if (recipients?.at(1)) {
       throw UnsupportedError(`Cannot create ${type} trigger for more than one recipient`);
     }
-    const [recipientId] = triggerInput.recipients;
-    const recipientUser = await internalLoadById<BasicStoreEntity>(context, user, recipientId, { type: ENTITY_TYPE_USER });
-    if (!recipientUser) {
-      throw UnsupportedError('Recipient user not found');
-    }
-    return recipientUser;
+    recipient = recipients.at(0) as string;
```

Comment:
You can avoid the as string

    recipient = recipients[0];

Reply:
This is to avoid Typescript error :   `Type 'undefined' is not assignable to type 'string'` ( `recipients.at(0)` can return undefined)

Reply:
I think you can try [recipient] = recipients;

Reply:
fixed( and no Typescript error)